sql/sql_string.h

```diff
@@ -473,6 +473,7 @@ class String
   }
   bool append(const String &s);
   bool append(const char *s);
+  bool append_uc(const char *s, const CHARSET_INFO *cs);
```

Comment:
`uc` is not a well-known acronym. A different name like `append_uppercased` would be better.